README.md

```diff
@@ -1,12 +1,14 @@
-<a href="https://istio.io/">
-    <img src="logo/istio-bluelogo-whitebackground-unframed.svg" alt="Istio logo" title="Istio" align="right" width="100" />
-</a>
-
 # Istio
```

Comment:
This section heading should come after the go report, godoc, and icon (at line 10).

Reply:
Moved.